### DISCUSSION TITLE
How to change to % instead of px in table cell

### DISCUSSION THREAD

**Discussion by dohomi** [ueberdosis/tiptap#5190] [General Questions]
Hey Guys

I would love to make adjustments to table cell widths, especially I want to enable responsive table cells. How could I move the pixel value to % value for the table widths?

Thanks

**Comment by nperez0111** [comment]
Our table implementation could use some love. Unfortunately we seem to be [parsing it as an int](https://github.com/ueberdosis/tiptap/blob/b941eea6daba09d48a5d18ccc1b9a1d84b2249dd/packages/extension-table-cell/src/table-cell.ts#L40) so it is not supported in other units than px in the default implementation

To get around this, it is probably easiest for you to extend our tableCell extension yourself and add it in yourself: https://tiptap.dev/docs/editor/guide/custom-extensions#extend-existing-extensions

**Reply by dohomi** [reply]
Thanks for your reply, I was already thinking that this might be the only way. I would like to just preset columns like:
2 columns with: 1/4 | 3/4
3 columns with: 1/4 | 2/4 | 1/4
as common preset. Now I try to find a way of:
1. set width to columns
2. generate the rows + columns with the desired widths per col on initiation. 
3. disable manual adjustment of the columns.

Will look deeper into this as soon I find the time

**Reply by nperez0111** [reply]
Sorry I couldn't be of more help!
